Relaxed PH spoke (#513)
Pyomo/mpi-sppy: doc/src/spokes.rst

The reduced cost spoke is equivalent to the Lagrangian spoke, except that it relaxes all 
integrality contraints in the subproblems. This enables the computation of reduced costs
for the nonanticipative variables, which can be used for provable bound tightening, which
is shared to all cylinders. The reduced cost can also be used for heuristic fixing in the

> hub via the reduced_cost_fixer.

Author: ```suggestion
hub via the reduced_cost_fixer extension.
```

cross scenario
^^^^^^^^^^^^^^

> Computed and passes cross scenario cuts.

Author: ```suggestion
Computes and passes cross scenario cuts.
```

relaxed_ph
^^^^^^^^^^

For S-MIPs, runs progressive hedging on the linear programming relaxation
of the scenario subproblems. Provides Ws and "relaxed nonants", the former
of which can be utilized for Lagrangian to compute lower bounds, and the later
of which can be utilized to inform fixings for the hub via the relaxed_ph_fixer

> extention. This method is effective when the subproblem linear programming

Author: ```suggestion
extention. This method can be effective when the subproblem linear programming
```
